R/filterResults.R

```diff
@@ -21,26 +21,26 @@ filter_seqs = function(x, ...) UseMethod("filter_seqs")
 #' @describeIn filter_seqs Method for primerTree objects
 #' @export
 filter_seqs.primerTree = function(x, min_length = 0, max_length = Inf, ...) {
-  #if(is.null(primerTreeObj$sequence)) {
+  #if(is.null(x$sequence)) {
   #  print "No sequences in primerTree object"
   #}
   
   # calculate how many removed and print out to screen
-  lengths <- unlist(lapply(primertree_obj$sequence, length))
+  lengths <- unlist(lapply(x$sequence, length))
```

Comment:
IIRC R 3.2 has a `lengths()` function to do this, so you could just call `lengths(x$sequence)`

For older R versions I would use `vapply(x$sequence, length, integer(1))` which is safer and more efficient.